cli/js/web/dom_types.d.ts

```diff
@@ -333,3 +333,13 @@ export interface ResponseConstructor {
   error(): Response;
   redirect(url: string, status?: number): Response;
 }
+
+export interface Storage {
```

Comment:
Do not add types here, implement from the global scope, we have too many problems with interfaces being mis-aligned and have been removing stuff from here.  See things like `Headers` and `ReadableStream` for examples.